PopPUNK/citation.py

```diff
@@ -45,12 +45,14 @@
 "pp-sketchlib version %(sketchlib_version)s (doi:%(sketchlib_doi)s) with " + \
 "sketch version %(sketchlib_hash)s, k-mer lengths %(kmin)s-%(kmax)s, a " + \
 "sketch size of %(sketch_size)s and %(seed_type)s seeds [6-8]. We assigned " + \
-"variable-length-k-mer clusters (VLKCs) using PopPUNK version %(poppunk_version)s " + \
-"(doi:%(poppunk_doi)s) by fitting a %(model_mode)s with %(model_options)s [1-5].\n"
+"isolates to %(cluster_type)s through variable-length-k-mer clustering (VLKC) with PopPUNK " + \
+"version %(poppunk_version)s (doi:%(poppunk_doi)s) by fitting a %(model_mode)s " + \
+"with %(model_options)s [1-5].\n"
 
 assign_methods = "We queried a database of %(number_samples)s isolates using pp-sketchlib version" + \
 " %(sketchlib_version)s (doi:%(sketchlib_doi)s) with sketch version %(sketchlib_hash)s, " + \
-"k-mer lengths %(kmin)s-%(kmax)s, a sketch size of %(sketch_size)s and %(seed_type)s seeds [6-8]. We assigned sequences to variable-length-k-mer clusters (VLKCs) using PopPUNK " + \
+"k-mer lengths %(kmin)s-%(kmax)s, a sketch size of %(sketch_size)s and %(seed_type)s seeds [6-8]. We " + \
+"sequence were classified using variable-length-k-mer clustering (VLKC) with PopPUNK " + \
```

Comment:
Think there might be a typo in this line?
'We sequence were classified using variable-length-k-mer clustering (VLKC) with PopPUNK'